### PR TITLE
README: Add modifier option to the VerticalResults portion of the doc…

### PR DESCRIPTION
**Pull request by alexisgrow**
Add modifier option to the VerticalResults portion of the documentation.

TEST=none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -494,6 +494,8 @@ ANSWERS.addComponent('VerticalResults', {
   maxNumberOfColumns: 1,
   // Optional, whether to display the total number of results, default true
   showResultCount: true,
+  // Optional, a modifier that will be appended to a class on the results list like this `yxt-Results--{modifier}`
+  modifier: true,
```
Comment (alexisgrow) [inline README.md]: Yikes this is a bad example. Updating.

Comment (nbramblett) [inline README.md]: Do we want this to default to true? That would modify the DOM technically, wouldn't it?

Comment (alexisgrow) [reply README.md]: Updated!

File: README.md [hunk 2]
```diff
@@ -494,6 +494,8 @@ ANSWERS.addComponent('VerticalResults', {
   maxNumberOfColumns: 1,
   // Optional, whether to display the total number of results, default true
   showResultCount: true,
+  // Optional, a modifier that will be appended to a class on the results list like this `yxt-Results--{modifier}`
+  modifier: 'Standard',
```
Comment (nbramblett) [inline README.md]: Similar question about defaulting to Standard. Is this a purely additive classname change?

Comment (alexisgrow) [reply README.md]: This isn't a default, this is an example, I will update it to empty.

**Review by nbramblett — APPROVED** [review]
LGTM!

**Pull request merged**